### PR TITLE
Feature/map menu v3

### PR DESCRIPTION
**Pull request by benkt2**
This is the correct map to merge - has the main map behaviour right but missing radio buttons on layers

### CODE REVIEW

File: app/javascript/MapPage.vue [hunk 1]
```diff
@@ -1,25 +1,70 @@
...
+	    required: true,
+	    type: Object},
+	selectedid: {
+            type: Number,
+            required: true},
+	},
+    
+    data() {
     return {
       mapboxToken: 'pk.eyJ1IjoibGV2aWF0aGFuczE3IiwiYSI6ImNpeDd5YWIzZTAwM3Myb29jaHNleW02YTgifQ.KOR1dSr7sTbWUtXw4V6tpA'
```
Comment (spencerldixon) [inline app/javascript/MapPage.vue]: Should this also be in the credentials?

Comment (benkt2) [reply app/javascript/MapPage.vue]: Probably. At the moment the API layer is basically a CnP of the json that was hardcoded in the vue. This is what we need to talk about in terms of separating which bits need to be per-layer and which bits are config

Comment (spencerldixon) [reply app/javascript/MapPage.vue]: Do you know what the scope of this token is? Is it read only?

Comment (spencerldixon) [reply app/javascript/MapPage.vue]: And do we have different keys for our different environments?

Comment (benkt2) [reply app/javascript/MapPage.vue]: This is a read-only token called 'NCFA Testing': there are others called things like pp-live-report but no other NCFA ones (yet). So we should create a prod one and kill this one when we've got it in secrets?

**Pull request merged**